### DISCUSSION TITLE
Release 2.0.0b1 - beta quality, for testing only

### DISCUSSION THREAD

**Discussion by ThomasWaldmann** [borgbackup/borg#6952] [General]
borgbackup 2.0.0b1 beta release - do not use this on production backup repositories.

Long changelog:
https://borgbackup.readthedocs.io/en/2.0.0b1/changes.html#version-2-0-0b1-2022-08-08

If you use pip to install this, use: pip install "borgbackup==2.0.0b1"

Docs: https://borgbackup.readthedocs.io/en/2.0.0b1/

<hr /><em>This discussion was created from the release <a href='https://github.com/borgbackup/borg/releases/tag/2.0.0b1'>Release 2.0.0b1 - beta quality, for testing only</a>.</em>

**Comment by DiagonalArg** [comment]
In [the docs](https://borgbackup.readthedocs.io/en/2.0.0b1/changes.html#version-2-0-0b1-2022-08-08) you say:

> differently than with borg 1.x you ONLY give the repo there, never a ::archive.

What does "there" refer to?  Where?

**Reply by ThomasWaldmann** [reply]
In borg 1.x there was a positional cli parameter usually referred to as `LOCATION` and you could give either `repo` there or `repo::archivename`. And the repo part was either a path or a ssh:// url or a scp style spec.

In borg 2.0 there will be a `--repository` (short `-r`) option to specify the repo **only**, you'll never append `::archivename` there.

The repo will be either a path or a url (no scp style any more).

If required, the archive name will often be given as a positional parameter (if usually only giving one archive name makes sense) or using `-a archivename_pattern` to match one or multiple archives.

This was done to simplify and disambiguate parsing of that stuff, mixing it all together as in borg 1.x was not a good idea. In some cases there were real problems (like our parser not parsing correctly), in other cases it was just unpretty and of course there is the general idea of not putting stuff together if it does not belong together.

**Comment by romario74** [comment]
There is no linuxold64 version anymore.
This will break (OL7|Centos7|RHEL7) support. Will you reconsider supporting them?
Big thx for this software.

**Reply by ThomasWaldmann** [reply]
I use different vagrant VMs to generate the binaries, using different releases of debian (see Vagrantfile).

Guess I removed the debian stretch VM because one of these:
- it is out of support from debian since quite a while
- maybe there were some requirements/dependency issues so it stopped working for borg2

I can have a look whether it can be revived / fixed somehow.

Another option for rhel7 users is of course to either continue using borg 1.2.x (or even 1.1.x) or to upgrade to some newer rhel release.

**Reply by ThomasWaldmann** [reply]
https://github.com/borgbackup/borg/commit/fc67453bf3e17000da10ba3a4dfb691f39294633 see there.

so, guess it could be re-added. of course that does not make the support from debian situation better, but we could offer the binary for whoever wants to use it nevertheless.

**Reply by ThomasWaldmann** [reply]
@romario74 no, we can't build borg2 on debian stretch. i tried, but stretch does not have libxxhash-dev which borg2 needs.

**Reply by romario74** [reply]
Merci for trying.
